internal/page/sign_out.go

```diff
@@ -24,6 +24,8 @@ func SignOut(logger Logger, sessionStore SessionStore, oneLoginClient OneLoginCl
 			endSessionURL = redirectURL
 		}
 
+		logger.InfoContext(r.Context(), "logout")
```

Comment:
Could it be useful to attach a session ID here to match the login event? Thinking specifically if we wanted/needed to track time active.

Reply:
session_id will be added here from the context, it's only on login that we need to manually add it